### PR TITLE
Update Reset Root Password to match the latest Citrix Doc

### PR DESCRIPTION
**Pull request by ManonMercier**
Signed-off-by: Manon Mercier <manon.mercier@vates.fr>

> Before submitting the pull request, you must agree with the following statements by checking both boxes with a 'x'.
> * [x] "I accept that my contribution is placed under the CC BY-SA 2.0 license [1]."
> * [x] "My contribution complies with the Developer Certificate of Origin [2]." 
>
> [1] https://creativecommons.org/licenses/by-sa/2.0/
> [2] https://xcp-ng.org/docs/contributing.html#developer-certificate-of-origin-dco

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -423,15 +423,16 @@ primitive p_iscsi_lun_1 iSCSILogicalUnit \
...
-Hi, this is a small trick I had to use once [(original article)](https://linuxconfig.org/how-to-reset-an-administrative-root-password-on-xenserver-7-linux)
-
-* Reboot your XenServer into Grub boot menu.
-* Use arrows keys to locate an appropriate XenServer boot menu entry and press **e** key to edit boot options.
-* Locate read-only parameter **ro** and replace it with **rw**. Furthermore, locate keyword **splash** and replace it with **init=/bin/bash**.
-* **Hit F10** to boot into single-mode
-* Once in single-mode use **passwd** command to reset your XenServer's root password
-* Reboot xenserver by entering the command **exec /usr/sbin/init**
-* If everything went well you should now be able to login with your new XenServer password.
+Hi, this is a small trick I had to use once [(original article)](https://support.citrix.com/article/CTX214360)
```
Comment (olivierlambert) [inline docs/troubleshooting.md:426]: We can probably rephrase this to be a bit more formal. Explaining the context, something like "If you need to reset your XCP-ng root password, you can follow this procedure".

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -423,15 +423,16 @@ primitive p_iscsi_lun_1 iSCSILogicalUnit \
...
+
+* Reboot your XCP-ng into Grub boot menu.
+* Select XCP-ng boot menu entry and press **e** key to edit boot options.
```
Comment (olivierlambert) [inline docs/troubleshooting.md:429]: Instead of **e** to emphasis on the key, we could use `e` instead.

File: docs/troubleshooting.md [hunk 3]
```diff
@@ -423,15 +423,16 @@ primitive p_iscsi_lun_1 iSCSILogicalUnit \
...
+* Locate read-only parameter **ro** and replace it with **rw init=/sysroot/bin/sh**.
```
Comment (olivierlambert) [inline docs/troubleshooting.md:430]: Same comment apply, replace all Markdown bold by code style (using backticks)

**Review by olivierlambert — CHANGES_REQUESTED** [review]

**Comment by olivierlambert** [comment]
Don't close the PR ;) You can make modification and re-push your code, this will solve the comments.

**Comment by stormi** [comment]
Closing the PR was needed to fix the DCO (they did not want to use git, but I think they'll need to in the end)

**Comment by stormi** [comment]
I fixed the DCO. @ManonMercier you can now handle the comments in a new commit (don't forget the DCO for that commit of course).

**Review by olivierlambert — APPROVED** [review]
Nice catch about the <kbd> thing, I wasn't aware of this!

File: docs/troubleshooting.md [hunk 4]
```diff
@@ -423,15 +423,16 @@ primitive p_iscsi_lun_1 iSCSILogicalUnit \
...
-Hi, this is a small trick I had to use once [(original article)](https://linuxconfig.org/how-to-reset-an-administrative-root-password-on-xenserver-7-linux)
-
-* Reboot your XenServer into Grub boot menu.
-* Use arrows keys to locate an appropriate XenServer boot menu entry and press **e** key to edit boot options.
-* Locate read-only parameter **ro** and replace it with **rw**. Furthermore, locate keyword **splash** and replace it with **init=/bin/bash**.
-* **Hit F10** to boot into single-mode
-* Once in single-mode use **passwd** command to reset your XenServer's root password
-* Reboot xenserver by entering the command **exec /usr/sbin/init**
-* If everything went well you should now be able to login with your new XenServer password.
+If you need to modify your XCP-ng root password, you may follow the steps below. The full prodecure can also be found on [this page](https://support.citrix.com/article/CTX214360).
```
Comment (stormi) [inline docs/troubleshooting.md:426]: What's a prodecure ? :D
